tests/v2/validation/nodescaling/scaling_node_driver_test.go

```diff
@@ -65,10 +65,10 @@ func (s *NodeScalingTestSuite) TestScalingNodePools() {
 		nodeRoles machinepools.NodeRoles
 		client    *rancher.Client
 	}{
-		{"Scaling control plane by 1", nodeRolesControlPlane, s.client},
-		{"Scaling etcd by 1", nodeRolesEtcd, s.client},
-		{"Scaling worker by 1", nodeRolesWorker, s.client},
-		{"Scaling worker by 2", nodeRolesTwoWorkers, s.client},
+		{"Scaling v2prov control plane by 1", nodeRolesControlPlane, s.client},
+		{"Scaling v2prov etcd by 1", nodeRolesEtcd, s.client},
+		{"Scaling v2prov worker by 1", nodeRolesWorker, s.client},
+		{"Scaling v2prov worker by 2", nodeRolesTwoWorkers, s.client},
```

Comment:
same question here. As none of our other qase tests have 'v2prov' as a cluster type, it may be confusing down the line.

Reply:
Resolved